### PR TITLE
Update icon set and improve paint-icon mixin

### PR DESCRIPTION
**Pull request by deioo**
:point_up:

Includes 4 more icons in the set and tweaks the mixin selectors

### CODE REVIEW

**Comment by carlosepp** [comment]
:+1:

File: components/_icon.scss [hunk 1]
```diff
@@ -55,34 +59,30 @@ $paint-icons: (
 ///     @include icon(dashboard);
 ///   }
 
-@mixin icon($name, $selector: '') {
+@mixin icon($name, $selector: null) {
```
Comment (carlosepp) [inline components/_icon.scss]: Is there a reason not to default this to :before always?, I find that in most cases I endup passing a before selector

Comment (deioo) [inline components/_icon.scss]: Yes, I guess we can refactor this once we switch to a single icon source, we'll have to replace some of the includes anyway.

Comment (carlosepp) [inline components/_icon.scss]: :cool: :100:

**Pull request merged**